src/Resources/CollectionRestResource.php

```diff
@@ -149,10 +166,11 @@ private function listItems(RestHandler $handler = null, $asSummary = false)
      * @param $handler
      * @return \stdClass
      */
-    protected function createCollectionResourceForItems($items, $from, $to, $handler)
+    protected function createCollectionResourceForItems($items, $from, $to, $count, $handler)
     {
         $resource = parent::get($handler);
```

Comment:
Should this call $this->get()? And I don't think the method needs $handler now, since get() doesn't need it.